main.go

```diff
@@ -415,7 +415,15 @@ func main() {
 			}
 
 			s := string(b)
-			s = strings.Replace(s, `"host":"openshift.io"`, `"host":"`+config.GetHTTPAddress()+`"`, -1)
+			// if set, use the deployments service URL for a proper host URL
+			// and fallback to the configured HTTP Address
+			serviceURL := config.GetDeploymentsServiceURL()
```

Comment:
Why not use something like https://github.com/fabric8-services/fabric8-wit/blob/master/rest/url.go#L15 instead?